src/AutorestSwift/View Models/KeyValueViewModel.swift

```diff
@@ -153,31 +153,31 @@ struct KeyValueViewModel {
         self.needDecodingInMethod = false
         self.constantValue = nil
     }
-}
 
-/**
- Convert the type into String format in Swift
- */
-private func convertValueToStringInSwift(type: AllSchemaTypes, value: String, key: String? = nil) -> String {
-    switch type {
-    case .string:
-        return "\(key ?? value)"
-    case .integer,
-         .number,
-         .boolean:
-        return "String(\(key ?? value))"
-    // For these types, a variable will be created in the method using the naming convention `{key|value}String`
-    case .date,
-         .unixTime,
-         .dateTime,
-         .byteArray:
-        return "\(key ?? value)String"
-    case .choice,
-         .sealedChoice:
-        return "\(value).rawValue"
-    case .array:
-        return "\(value).map { String($0) }.joined(separator: \",\") "
-    default:
-        return "\(value)"
+    /**
+     Convert the type into String format in Swift
+     */
+    private static func formatValueForType(type: AllSchemaTypes, value: String, key: String? = nil) -> String {
```

Comment:
The double declaration of type should be avoided. The "Swifty" way:
`formatValue(forType type: ...)`